parmesan/distributions.py

```diff
@@ -1,13 +1,13 @@
 import math
 import theano.tensor as T
+c = - 0.5 * math.log(2*math.pi)
 
 
-c = - 0.5 * math.log(2*math.pi)
 def log_normal(x, mean, sd):
     """
     Compute log pdf of a Gaussian distribution with diagonal covariance, at values x.
 
-        .. math:: \log p(x) = \log \mathcal{N}(x; \mu, \sigma^2I)
+        .. math:: \log p(x) = \log \mathcal{N}(x; \mean, \sigma^2I)
```

Comment:
The `\mu` in the docstrings was supposed to be a LaTeX symbol.

I'd either leave it like it was or change to something like
```
    .. math:: \log p(x) = \log \mathcal{N}(x; \mathtt{mean}, \exp \mathtt{log_var} I)
```